protocol/app/app_test.go

```diff
@@ -150,11 +150,11 @@ func TestSimulationManager(t *testing.T) {
 }
 
 func TestUpgrades(t *testing.T) {
-	require.Len(t, app.Upgrades, 0, "Expected no Upgrades")
+	require.Len(t, app.Upgrades, 1, "Expected 1 Upgrades")
 }
 
 func TestForks(t *testing.T) {
-	require.Len(t, app.Forks, 0, "Expected no Forks")
```

Comment:
These are low value unit tests in my opinion and are just going to delay things in cases like this. 

I am going to remove them on main if there is no objection.